jmi_erpnext/fixtures/print_format.json

```diff
@@ -49,9 +49,9 @@
t\t<div class=\"col-xs-9 pull-left\" >\n\t\t\t\t<p><b>{{ doc.customer }}</b></p>\n\t\t\t\t<p> {{ doc.shipping_address }}</p>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n\t<hr>\n\n\t<div class=\"row\">\n\t\t<div class=\"col-xs-4\" >\n\t\t\t<p><b style=\"font-size:14px;\">Customer ID:&nbsp;</b>{{ doc.jmi_customer_id or None}} </p>\n\t\t</div>\n\t\t<div class=\"col-xs-4\">\n\t\t\t<p><b style=\"font-size:14px;\">Sales Rep:&nbsp;</b>{{ frappe.get_doc(\"User\",frappe.session.user).full_name }}</p>\n\t\t</div>\n\t\t<div class=\"col-xs-4\">\n\t\t\t<p><b style=\"font-size:14px;\">PO No.:&nbsp;</b>{{ doc.po_no or None}}</p>\n\t\t</div>\n\t</div>\n\n\t<table class=\"table table-condensed table-bordered\"> \n\t\t<thead>\n\t\t\t<tr>\n\t\t\t\t<th class=\"col-xs-1\">Sr.No</th>\n\t\t\t\t<th class=\"col-xs-2\">Item Code </th>\n\t\t\t\t<th class=\"col-xs-4\">Description</th>\n\t\t\t\t<th class=\"col-xs-1 text-center\">Qty</th>\n\t\t\t\t<th class=\"col-xs-2\" style=\"text-align:right;\">Price  </th>\n\t\t\t\t<th class=\"col-xs-2\" style=\"text-align:right;\">Total</th>\n\t\t\t</tr>\n\t\t</thead>\n\t\t<tbody>\n\t\t\t{% for row in doc.items %}\n\t\t\t\t<tr>\n\t\t\t\t\t<td class=\"col-xs-1\">{{ row.idx }}</td>\n\t\t\t\t\t<td class=\"col-xs-2\">{{ row.item_code }}</td>\n\t\t\t\t\t<td class=\"col-xs-4\">{{ row.description }}</td>\t\t\n\t\t\t\t\t<td class=\"col-xs-1\">\n\t\t\t\t\t\t<span class=\"pull-left\">{{ frappe.format_value((row.qty), {\"fieldtype\":\"Float\"}) }}</span>\n\t\t\t\t\t\t<span class=\"pull-right\">{{ row.uom }}</span>\n\t\t\t\t\t</td>\n\t\t\t\t\t<td class=\"col-xs-2\" style=\"text-align:right\">{{ row.get_formatted(\"rate\") }}\n\t\t\t\t</td>\t\t\t\t\t\n\t\t\t\t\t<td class=\"col-xs-2\" style=\"text-align:right\">\n\t\t\t\t\t\t{{ row.get_formatted(\"amount\") }}\n\t\t\t\t\t</td>\t\t\t\t\t\t\t\t\n\t\t\t\t</tr>\n\t\t\t{% endfor %}\n\t\t\t<tr>\n\t\t\t\t<td colspan=\"4\" style=\"text-align:right;\">\n\t\t\t\t\t<p>Net Total:</p>\n\t\t\t\t\t{% for tax in doc.taxes %}\n\t\t\t\t\t\t{% if (tax.charge_type!=\"Actual\") %}\n\t\t\t\t\t\t\t<p>{{ tax.description }}:</p>\n\t\t\t\t\t\t{% endif %}\n\t\t\t\t\t{% endfor %}\n\t\t\t\t\t<p>Total Due:</p>\n\t\t\t\t\t<p> Change Amount: </p>\n\t\t\t\t</td>\n\t\t\t\t<td colspan=\"1\">\n\t\t\t\t\t<p style=\"text-align:right\">{{ doc.get_formatted(\"total\") }}</p>\t\n\t\t\t\t\t{% for tax in doc.taxes %}\n\t\t\t\t\t\t{% if (tax.charge_type!=\"Actual\") %}\n\t\t\t\t\t\t\t<p style=\"text-align:right\">\n\t\t\t\t\t\t\t\t{{ tax.get_formatted(\"tax_amount\", doc) }}\n\t\t\t\t\t\t\t</p>\n\t\t\t\t\t\t{% endif %}\n\t\t\t\t\t{% endfor %}\n\t\t\t\t\t<p style=\"text-align:right\">{{ doc.get_formatted(\"grand_total\") }}</p>\n\t\t\t\t\t<p style=\"text-align:right\">{{ doc.get_formatted(\"change_amount\") }} </p>\n\t\t\t\t\t\n\t\t\t\t</td>\n\t\t\t</tr>\n\t\t</tbody>\n\t</table>\n\t<div id=\"footer-html\">\n\t\t{% if doc.letter_head %}\n\t\t\t<div class=\"letter-head-footer\">\n\t\t\t\t{{ footer }}\n\t\t\t</div>\n\t\t{% endif %}\n\t</div>\n</div>\n", 
```

Comment:
Any changes in print format?

Reply:
Yes, has the 'change amount' field in print format